sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/ParDo.java

```diff
@@ -24,8 +24,9 @@
 import java.lang.reflect.Type;
 import java.util.Arrays;
 import java.util.Collections;
-import java.util.List;
```

Comment:
We need to validate that the types of the parameters match the types of the side input. Possibly we should do this in the validate() method - there we can look up the elementT of each parameter and make sure it matches the elementT of the PCollectionView.